main.js

```diff
@@ -1,173 +1,249 @@
-function getJIRAFeed(callback, errorCallback){
-    var user = document.getElementById("user").value;
-    if(user == undefined) return;
-    
-    var url = "https://jira.secondlife.com/activity?maxResults=50&streams=user+IS+"+user+"&providers=issues";
-    make_request(url, "").then(function(response) {
-      // empty response type allows the request.responseXML property to be returned in the makeRequest call
-      callback(url, response);
-    }, errorCallback);
+const STATUS_BASE_URL = `https://jira.secondlife.com/rest/api/2/search?jql=`;
+const PROJECT_URL = `https://jira.secondlife.com/rest/api/2/project/SUN`;
+const FEED_BASE_URL = `https://jira.secondlife.com/activity?maxResults=50`;
+
+// Setup
+document.addEventListener('DOMContentLoaded', async function() {
+	try {
+		// if logged in, setup listeners
+		await checkProjectExists();
+		//load saved options
+		await loadOptions();
```

Comment:
I like this await flow, however, you've slowed down load with the second `await`. With more time, what would you do to address this UX change?